HospitalSystem.ConsoleUI/Doctor/StartCheckupUI.cs

```diff
@@ -0,0 +1,290 @@
+using MongoDB.Bson;
+using MongoDB.Driver;
+using HospitalSystem.Core.Utils;
+using HospitalSystem.Core;
+
+namespace HospitalSystem.ConsoleUI;
+
+public class StartCheckupUI : DoctorCheckupsUI
+{
+    Checkup Checkup;
+    public StartCheckupUI(Hospital hospital, User user, Checkup checkup) : base(hospital, user) { 
+        Checkup = checkup;
+    }
+    public override void Start()
+    {
+        Console.Clear();
+        bool quit = false;
+        Patient patient =  ShowPatientInfo(Checkup);
+        Console.WriteLine("\n\nCheckup started.\n");
+        while (!quit)
+        {
+            Console.Write("\nCheckup options:\n\n1. Add Anamnesis\n2. Edit Medical Record\n3. Write referral\n4. End Checkup\n\n>>");
+            var input = Console.ReadLine();
+            switch (input)
+            {
+                case "1":
+                {
+                    addAnamnesis(patient);
+                    break;
+                }
+                case "2":
+                {
+                    EditMedicalRecord(patient);
+                    break;
+                }
+                case "3":
+                {
+                    WriteReferral(patient);
+                    break;
+                }
+                case "4":
+                {
+                    EquipmentStateUpdate(Checkup);
+                    _hospital.AppointmentService.FinishCheckup(Checkup);
+                    quit = true;
+                    break;
+                }
+                default:
+                {
+                    Console.WriteLine("Wrong input. Please enter one of the options above.");
+                    break;
+                }
+            } 
+        }  
+    }
+
+    public void addAnamnesis(Patient patient)
+    {
+        Console.Write("\nEnter Anamnesis >> ");
+        String? anamnesis = Console.ReadLine();
+
+        patient.MedicalRecord.AnamnesisHistory.Add(anamnesis);
+        _hospital.PatientService.AddOrUpdatePatient(patient);
+
+        Checkup.Anamnesis = anamnesis;
+        _hospital.AppointmentService.UpsertCheckup(Checkup);
+
+        Console.Write("\nDo you want to add a prescription? [y/n] >> ");
+        string choice = ReadSanitizedLine();
+        if (choice == "y")
+        {
+            PrescriptionMenu(patient);
+        }
+    }
+
+    public void EditMedicalRecord(Patient patient)
+    {
+        Console.Write("\nEdit options:\n1. Edit height\n2. Edit weight\n3. Add allergy\n4. Back\n");
+        Console.Write(">>");
+        var input = Console.ReadLine();
```

Comment:
Why do you just dont use ReadSanitizedLine() instead of Console.Readline(), just a tip